reference-app/app.js

```diff
@@ -180,6 +180,19 @@ mainModule.controller('MainCtrl', ['$scope', '$http', '$q', 'remote', 'config',
         return true;
     }
 
+    $scope.getResourceStyle = function(resource) {
+        let style = {};
+
+        if(resource.rt[0] === 'oic.r.colour.rgb') {
+            let rgb = resource.rgbvalue;
+            if(rgb && rgb[0] !== null && rgb[1] !== null && rgb[2] !== null) {
+                style['background-color'] = `rgb(${Math.round(rgb[0])},${Math.round(rgb[1])},${Math.round(rgb[2])})`;
+            }
```

Comment:
else what happens ? error logged somewhere? is this even a valid scenario where these values will be null? if not should we warn?